### ISSUE TITLE
Unable to lanch Code after upgrade to 1.57

### ISSUE THREAD

**Issue by erikfroseth** [microsoft/vscode#126019]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 
1.57.0
b4c1bd0a9b03c749ea011b06c6d2676c8091a70c
x64

- OS Version:
Ubuntu 20.04.2 LTS

Steps to Reproduce:

1. Upgrade vscode to latest version using apt
2. Start vscode either by clicking the icon in Gnome, or writing "code" in the terminal

I know this bug report doesn't help much, but I do not know what else to include. I did "apt update" this morning, and vscode stopped working after this. When writing "code" in my terminal, nothing happens. No output, nothing. The same has happened to a colleague with the same OS/machine. If there is anything more I should add (logs, etc), I would be more than happy to include it.

I noticed that there are some bug reports on the settings file being corrupted, but that does not seem to have happened to me.

**Comment by bpasero** [comment]
Does it work when you run `code --no-sandbox`?

**Comment by erikfroseth** [comment]
@bpasero Yes, that helps!

**Comment by bpasero** [comment]
@erikfroseth how did you install VSCode? Snap?

**Comment by erikfroseth** [comment]
I installed it using this guide: https://code.visualstudio.com/docs/setup/linux#_debian-and-ubuntu-based-distributions

Downloaded the .deb and did the following steps:
```
sudo apt install apt-transport-https
sudo apt update
sudo apt install code # or code-insiders
```

**Comment by bpasero** [comment]
Any chance you are running `code` as `sudo`?

**Comment by erikfroseth** [comment]
No, I'm not running code as sudo. If I try running it as sudo, I get a warning saying `You are trying to start Visual Studio Code as a super user ...`

**Comment by bpasero** [comment]
How about running with `--disable-setuid-sandbox`?

**Comment by erikfroseth** [comment]
It still does not start. It gives me a warning, though: `Warning: 'disable-setuid-sanbox' is not in the list of known options, but still passed to Electron/Chromium.`

However, I see that entering "code" in my terminal does start some processes that never dies:

```
erikfroseth@erik-desktop:~$ ps -A | grep code
erikfroseth@erik-desktop:~$ code
erikfroseth@erik-desktop:~$ ps -A | grep code
  43167 ?        00:00:00 code
  43170 ?        00:00:00 code
  43171 ?        00:00:00 code
```

**Comment by Ekmansoft** [comment]
I have the same problem

**Comment by deepak1556** [comment]
@erikfroseth can you make sure to kill all `code` processes and then try https://github.com/microsoft/vscode/issues/126019#issuecomment-859413252

**Comment by erikfroseth** [comment]
@deepak1556 Still the same problem :-(

**Comment by Ekmansoft** [comment]
> @deepak1556 Still the same problem :-(

Me too

**Comment by Ekmansoft** [comment]
Killing the processes worked fine, and now no more processes appears, but still no sign of vscode starting

**Comment by bpasero** [comment]
We will push https://github.com/microsoft/vscode/issues/126027 for May recovery which will restore `--no-sandbox`.

**Comment by erikfroseth** [comment]
@bpasero Big kudos to you and the team for super fast response! Thank you very much for your help
